ports/kdsoap/portfile.cmake

```
@@ -33,3 +33,18 @@ file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.txt")
+file(
+    COPY
+        "${SOURCE_PATH}/LICENSES/BSD-3-Clause.txt"
+        "${SOURCE_PATH}/LICENSES/GPL-2.0-only.txt"
+        "${SOURCE_PATH}/LICENSES/LicenseRef-Microsoft.txt"
+        "${SOURCE_PATH}/LICENSES/LicenseRef-Novell.txt"
+        "${SOURCE_PATH}/LICENSES/LicenseRef-OASIS.txt"
+        "${SOURCE_PATH}/LICENSES/LicenseRef-SportingExchange.txt"
+        "${SOURCE_PATH}/LICENSES/MIT.txt"
+        "${SOURCE_PATH}/LICENSES/W3C.txt"
+    DESTINATION
+        "${CURRENT_PACKAGES_DIR}/share/${PORT}/LICENSES/"
+)

```

I would probably use this, and maybe add a COMMENT:
```suggestion
vcpkg_install_copyright(
    FILE_LIST
        "${SOURCE_PATH}/LICENSE.txt"
        "${SOURCE_PATH}/LICENSES/BSD-3-Clause.txt"
        "${SOURCE_PATH}/LICENSES/GPL-2.0-only.txt"
        "${SOURCE_PATH}/LICENSES/LicenseRef-Microsoft.txt"
        "${SOURCE_PATH}/LICENSES/LicenseRef-Novell.txt"
        "${SOURCE_PATH}/LICENSES/LicenseRef-OASIS.txt"
        "${SOURCE_PATH}/LICENSES/LicenseRef-SportingExchange.txt"
        "${SOURCE_PATH}/LICENSES/MIT.txt"
        "${SOURCE_PATH}/LICENSES/W3C.txt"
)
```

---

This will include all files in the copyright file, though, but the LICENSE.txt file contains "See the full license texts in the LICENSES folder". Is this ok?

---

Yes, it`s ok.

---

> This will include all files in the copyright file, though, but the LICENSE.txt file contains "See the full license texts in the LICENSES folder". Is this ok?

This is something which I would address by the COMMENT, or by direct modification of that phrase. 

The single `copyright` file is what it makes it easy to deploy the texts with the binaries. Having to handle individual extra folders is inviting trouble. That's why this maintainer function handling file lists was added AFAICT.

---

Done, I added a patch to replace

```diff
-See the full license texts in the LICENSES folder.
+See the full license texts provided below in this file.
```

and included all files in the copyright file. @dg0yt Does this look OK now, or do you think a COMMENT is still necessary?